packages/dd-trace/test/plugins/agent.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
     agent = express()
     agent.use(bodyParser.raw({ limit: Infinity, type: 'application/msgpack' }))
     agent.use((req, res, next) => {
-      if (req.body.length === 0) return res.status(200).send()
+      if (!req.body.length) return res.status(200).send()
```

Comment:
Why this change?

Reply:
`bodyParser` default to an empty object when the content type didn't match, which doesn't have a `length` property.